### PR TITLE
prettified example

### PR DESCRIPTION
**Pull request by thilob**

### CODE REVIEW

File: example.html [hunk 1]
```diff
@@ -1,14 +1,12 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
```
Comment (justpilot) [inline example.html:5]: SOLIDUS deprecated

File: example.html [hunk 2]
```diff
@@ -1,14 +1,12 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Aviation Functions Library</title>
-    <link rel="icon" href="data:;base64,iVBORw0KGgo=">
+    <link rel="icon" href="data:;base64,iVBORw0KGgo=" />
```
Comment (justpilot) [inline example.html:8]: SOLIDUS deprecated

File: example.html [hunk 3]
```diff
@@ -1,14 +1,12 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (justpilot) [inline example.html:6]: SOLIDUS deprecated

**Review by justpilot — CHANGES_REQUESTED** [review]
Please remove the SOLIDUS

**Comment by thilob** [comment]
removed them

**Pull request merged**